essay.tex

```diff
@@ -27,19 +27,32 @@
+\section{Obstacles to Efficient Meetings}
+
+One of the keys to an efficient daily meeting is to complete the meeting at the right time of day for the right amount of time in order to be productive by not wasting time. In a case study I researched \cite{Obstacles} it was found that there were many temporal obstacles to an efficient meeting. They observed that people often arrived late to meetings which meant the meeting would not start on time. This means that time is wasted for those who arrive on time. It was also found that many of the meetings in the case study lasted longer than 15 minutes. This was usually caused by people doing things like doodling on their notepad or using their phones. It was found that meetings lasted between 7 and 45 minutes with a mean of 22 minutes, which meant that the cost in time of the meeting was 47 percent more than planned. In addition to these obstacles, the meetings were not always at the same time of day. In cases where people have their daily meeting at the start of each day this is an issue. This is because team members wouldn't usually start tasks from the backlog until after the daily meeting, so if the daily meeting doesn't always happen at the start of day a lot of time will be wasted. Along with all of these temporal obstacles, research suggests that many team members perceived the meeting as an interruption of their work flow and would prefer fewer meetings. One paper discussed how having meetings every day can be bad in some cases. For example in cases where employees are assigned to several teams at the same time, daily meetings in each of the teams would lead to severe constraints on behalf of those employees \cite{ScrumEveryDay}.
+
+\section{Solutions}
+
+As you can see there are many common obstacles related to time preventing daily scrum meetings from being as productive as possible. I found a number of possible solutions to each of the obstacles. To make sure that meetings start on time, and at the same time every day, an alarm could be used to signal the start of the meeting \cite{Obstacles}.  A timer could be used to make sure that the meeting lasts 15 minutes long, it should be visible to every member in the meeting so they know how long is left. Many team members perceived the meetings as an interruption to their work flow, this was most likely due to how long the meetings were taking (up to 45 minutes in some cases), so if the previous solutions were implemented this might not be a problem anymore. This would mean the daily scrum meeting would still be used as intended, on a daily basis.    
```

Comment:
The core of your argument seems to rely on just one case study: `\cite{Obstacles}`. Perhaps you could extend this with some material from GDC Vault? There are some examples of good practice in the Guild Wars 2 talk, for example.